src/app/config/resolve_addr.c

```diff
@@ -596,27 +593,44 @@ resolve_my_address_v4(int warn_severity, const or_options_t *options,
 /** Return true iff <b>addr</b> is judged to be on the same network as us, or
  * on a private network.
  */
-MOCK_IMPL(int,
-is_local_addr, (const tor_addr_t *addr))
+MOCK_IMPL(bool,
+resolved_addr_is_local, (const tor_addr_t *addr))
```

Comment:
True... I just tried to match the file namespace and the file is called `resolved_addr.c` ... honestly all this might just be a global bad name because the functions in there are not only resolving...

This function goal is actually to make sure the given address is "local to us" or not. But it kind of does more with this "is internal"... but apart from this I'm wondering if it should be entirely renamed to something that reflects that like:

- `is_local_to_my_address()`
- `is_local_to_me()`
...

Thoughts? Now is a good time to bikeshed and decide since it is totally changed and one commit away from a rename.

Reply:
`is_local_to_me()` seems good, but only if it is looking at the actual address we're going to consider ours. If it's looking at something else (like the resolved address but not the interface address, for instance), it should say is_local_to_resolve_addr() or something like that, I think.  Do you agree?

Reply:
That function is taking an address, bigger use case is channetls for inbound connections, and make sure it is not in our subnet. If it is, it flags it "local".

So that function returns true is any address given is local to us. Lets used `is_local_to_resolve_addr()`.

Commit 2e951065ca74f2cc